client/core/src/com/cyberbot/bomberman/screens/lobby/LobbyLayout.java

```diff
@@ -117,35 +113,57 @@ public void clicked(InputEvent event, float x, float y) {
         });
     }
 
-    //TODO use public addPlayer api in recieving server request
-    public void addPlayer(String playerName) {
-        for (int i = 0; i < 4; i++) {
-            if (playerEmpty[i]) {
-                playerEmpty[i] = false;
-                playerLabels[i].setText(playerName);
-                return;
-            }
-        }
-        throw new ArrayIndexOutOfBoundsException("Too many players added");
-    }
-
-    public void addPlayer(String playername, int number){
-        playerEmpty[number] = false;
+    public void addPlayer(String playername, int number) {
         playerLabels[number].setText(playername);
     }
 
     public void removePlayer(int number) {
-        playerEmpty[number] = true;
         playerLabels[number].setText("Empty");
     }
```

Comment:
Those methods seem useless, as we'll usually update from the full state, not calculate the delta.